dshr/dshr_mod.F90

```diff
@@ -429,8 +432,7 @@ subroutine dshr_mesh_create(filename, scol_mode, scol_lon, scol_lat, &
     real(r8)          , intent(inout) :: scol_lon
     real(r8)          , intent(inout) :: scol_lat
     character(len=*)  , intent(in)    :: compname
-    integer           , intent(in)    :: my_task
-    integer           , intent(in)    :: master_task
+    logical           , intent(in)    :: my_task
```

Comment:
Please don't call this my_task - it should be masterproc shouldn't it?

Reply:
Looks like an error at line 683

Reply:
No - it should not be masterproc. It should only be my_task. Please see the above comment.

Reply:
There is no error at line 683.

Reply:
If it should be my_task then it is not a logical is it?

Reply:
No my_task is an integer. In looking at the code this looks correct. We need to look at the latest code I pushed back not the snipped above.